mmv1/third_party/terraform/services/compute/resource_compute_region_security_policy_test.go.tmpl

```diff
@@ -799,26 +852,52 @@ func testAccComputeRegionSecurityPolicy_withNetworkMatch(context map[string]inte
 
 func testAccComputeRegionSecurityPolicy_withNetworkMatch_update(context map[string]interface{}) string {
 	return acctest.Nprintf(`
+
+	resource "google_project" "project" {
+		name = "tf-region-ca-rules-%{random_suffix}"
+		project_id = "tf-region-ca-rules-%{random_suffix}"
```

Comment:
Needs a tf-test prefix here to be sweepable. I don't remember offhand if this has a short length limit.

```suggestion
		project_id = "tf-test-region-ca-rules-%{random_suffix}"
```